.github/workflows/python-package.yml

```diff
@@ -0,0 +1,35 @@
+name: Python Package
+
+on:
+  pull_request:
+    types: [opened, edited]
```

Comment:
Not sure this is required. But, we do want to limit the workflow to PRs with `main` as the target branch

Reply:
This is something along the lines of what we want:
```yml
on:
  # Trigger the workflow on push or pull request,
  # but only for the main branch
  pull_request:
    branches:
      - main
    types: [opened, edited]

```